content/response-to-reviews/therapeutics-v1.md

```diff
@@ -0,0 +1,87 @@
+Much of the original text, where individual papers are critiqued, has been moved to an appendix.
+We hope this will provide a broad view of the current state of therapeutic development for most readers, with in-depth critique available for readers who would like to investigate a particular therapeutic further.
+**
+
+1.2 Too much text discussing failed approaches and limitations of clinical trials. A considerable amount of text is spent on chloroquine, despite the wide consensus that it does not have an effect. In contrast, promising approaches like monoclonal antibodies are given just a few short paragraphs. I also felt that the in-depth criticisms of clinical trial design were unnecessary and distracted from the main message of each section. They are certainly valid points, but it would be great to get the conclusion across in a more concise manner, while providing references that enable interested readers to dive deeper into each study's strengths and weaknesses.
+
+**
+We hope that the new structure serves to address most of these concerns.
+The detailed critiques have been moved to the appendix, and the narratives surrounding failed approaches has been streamlined so as not to over-emphasize them.
+**
+
+It also seems dangerous to infer researcher biases like p-hacking without more definitive data that support these potentially damaging claims.
+
+**
+The sentence in question has been modified to read: "Additionally, disparities between the pre-registered and published protocols raises concerns about experimental design."
+**
+
+1.3 This article as presented seems premature. I was especially confused by the statement in the abstract that "this review will be updated as progress is made." Is that even possible at mSystems? 
+
+**
+The project is built on Manubot, a relatively new resource for collaborative writing that allow for version control and continual editing.
+An up-to-date version of the manuscript is available online at all times, but _mSystems_ has agreed to review a version of record and to allow the submission of updates as more evidence becomes available over time, in the style of a living review.
+**
+
+More importantly, the conclusion for all but 1-2 therapeutics is that more data is needed, leaving me wondering what the value of this review is at time present. One useful contribution would be to rank the promise of each approach. Each has issues, but which ones are worth investing more resources in?
+
+**
+We hope that the recontextualization of the content serves to address the concerns related to the usefulness of the project -- the goal is to organize the available evidence to allow scientists to get or stay up to speed on the status of these therapeutics without needing to dive independently into the exceptionally large body of literature.
```

Comment:
To do: integrate the rankings from https://cdcn.org/corona-data-viewer/ into the table and note that change here